integration-tests/helpers/draft-order-seeder.js

```diff
@@ -99,22 +126,20 @@ module.exports = async (dataSource, data = {}) => {
       },
     ],
   })
+  await manager.save(pv2)
 
-  const ma = manager.create(MoneyAmount, {
-    variant_id: "test-variant",
+  await manager.insert(MoneyAmount, {
+    id: "test-price-2",
     currency_code: "usd",
-    amount: 8000,
+    amount: 10000,
```

Comment:
question: Why did we changed the amount here?

Reply:
I didn't it's a git-diff messing up, product-variant-2 has historically had a price of 10000 and product-variant-1 has had 8000 which is still the case after this change. It's just github messing up what has actually been changed